File: java/src/estruturas/repetidas/Tabuada.java
package estruturas.repetidas;

import java.util.Scanner;

/**
 * Problema "tabuada"
 * Ler um número inteiro N, daí mostrar na tela a tabuada de N para 1 a 10.
 */
public class Tabuada {
  /**
   * The entry point of application.
   *
   * @param args the input arguments
   */
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);

    System.out.println("Deseja a tabuada para qual valor? ");

    final int n = sc.nextInt();

    for (int i = 0; i <= 10; i++) {
      System.out.printf("%d x %d = %d%n", i, n, i * n);
    }

    sc.close();
  }
}

File: python/src/estruturas/repetitivas/tabuada.py
""" Problema "tabuada"
 * Ler um número inteiro N, daí mostrar na tela a tabuada de N para 1 a 10.
"""


def tabuada() -> None:
    n = int(input("Deseja a tabuada para qual valor? "))

    for i in range(11):
        print(f"{i:2} x {n} = {i * n}")


if __name__ == "__main__":
    tabuada()

